config.js

```diff
@@ -11,12 +11,13 @@ module.exports = {
         'cm.js'
     ]
   , errorCountThreshold        : 10
-  , maxGzipped                 : 5 * 1024
+    // normal limit is 5kb, but strictly enforce 6kb
+  , maxGzipped                 : 6 * 1024
```

Comment:
seems like a better and more maintainable solution than `sizeExceptions` - just use discretion in accepting PRs